Source/WebCore/html/HTMLSummaryElement.cpp

```diff
@@ -111,7 +111,7 @@ int HTMLSummaryElement::defaultTabIndex() const
 
 bool HTMLSummaryElement::supportsFocus() const
 {
-    return isActiveSummary();
+    return isActiveSummary() || HTMLElement::supportsFocus();
```

Comment:
That's such an obvious bug too!

Reply:
I do recall fixing similar for another element and did some local testing and viola and it fixed failing tests. So win win!!